bridges/SymfonyCastsBridge.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+class SymfonyCastsBridge extends BridgeAbstract {
+	const NAME = 'SymfonyCasts Bridge';
+	const URI = 'https://symfonycasts.com/';
+	const DESCRIPTION = 'Follow new updates on symfonycasts.com';
+	const MAINTAINER = 'No maintainer';
+	const CACHE_TIMEOUT = 3600;
+
+	public function collectData() {
+		$html = getSimpleHTMLDOM('https://symfonycasts.com/updates/find');
+		$dives = $html->find('div');
+
+		/* @var simple_html_dom $div */
+		foreach ($dives as $div) {
+			//var_dump($div);die();
+			//dump_html_tree($div);die();
+			$id = $div->getAttribute('data-mark-update-id-value');
+			$type = $div->find('h5');
+			if(is_array($type)) {
+				$type = $type[0];
+			}
```

Comment:
This `is_array` is unnecessarily. You can just use `$div->find('h5', 0)` to get the `h5`,

```suggestion
			$type = $div->find('h5', 0);
```